lib/modules/dosomething/dosomething_reportback/dosomething_reportback.admin.inc

```diff
@@ -169,9 +169,7 @@ function dosomething_reportback_files_form($form, &$form_state, $status = NULL,
 
   foreach ($result as $record) {
     $rb_path = 'admin/reportback/' . $record->rbid;
-    // @TODO: temporarily disable pre-approved to unblock campaigns team from reviewing reportbacks.
-    // Refs #4797
-    // $status = 'approved';
+    $status = 'approved';
```

Comment:
yeah, pre-approved wasn't the thing that was causing the illegal choice error. 
it causes things to be approved that aren't actually being reviewed.